doc/classes/AudioStreamPlayer2D.xml

```diff
@@ -1,10 +1,13 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AudioStreamPlayer2D" inherits="Node2D" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Plays audio in 2D.
 	</brief_description>
 	<description>
+		Plays audio that dampens with distance of the object to screen center.
```

Comment:
Ref: style guide "avoid milk of the cow"

Should be "dampens with the object's distance *from* screen center"